packages/bpk-component-card/src/BpkCardWrapper.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import React, { type Node } from 'react';
+import { borderSizeLg } from '@skyscanner/bpk-foundations-web/tokens/base.es6';
+
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkCardWrapper.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  card: Node,
+  className: ?string,
+  backgroundColor: string,
+  header: Node,
+};
+
+const BpkCardWrapper = (props: Props) => {
+  const { backgroundColor, card, className, header } = props;
+
+  const classNames = getClassName('bpk-card-wrapper', className);
+
+  return (
+    <div
+      className={classNames}
+      style={{
+        boxShadow: `0 0 0 2px ${backgroundColor}`,
```

Comment:
~might be good to use the same method as below and set the boxShadow in CSS rather than doing it here?~ ah wait just realised we do so I guess we don't need this?
also wondering if we don't already have a border mixin which supports this size?

Reply:
Agree with you, move it into scss might would be better. For border mixins, i'd prefer use spacing mixins as mixed `$bpk-border-size-lg` and `bpk-spacing-sm()` here seems not quite pretty? On the other hand, we need this for card wrapper box shadow effect.

Reply:
It seems would result in some problem `box-shadow ` always comes in front of `@include` due to pre commit check and make box shadow not work.

Reply:
Oh, i realize it can use `@include bpk-border-lg`, it works now.